### PR TITLE
Update init-containers.md

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
image
image

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/init-containers.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ Here are some ideas for how to use init containers:
 
 * 在启动应用容器之前等一段时间，使用类似命令：
 
-        `sleep 60`
```
Comment (tanjunchen) [inline content/zh/docs/concepts/workloads/pods/init-containers.md:138]: this should be not changed.

File: content/zh/docs/concepts/workloads/pods/init-containers.md [hunk 2]
```diff
@@ -404,7 +404,7 @@ Pod level control groups (cgroups) are based on the effective Pod request and li
 给定Init 容器的执行顺序下，资源使用适用于如下规则：
 
 * 所有 Init 容器上定义的任何特定资源的 limit 或 request 的最大值，作为 Pod *有效初始 request/limit*
-* Pod 对资源的 *有效 limit/request * 是如下两者的较大者：
+* Pod 对资源的 *有效 limit/request* 是如下两者的较大者：
```
Comment (tanjunchen) [inline content/zh/docs/concepts/workloads/pods/init-containers.md:407]: this is good

File: content/zh/docs/concepts/workloads/pods/init-containers.md (hunk 1 above)
Comment (ZP-AlwaysWin) [reply content/zh/docs/concepts/workloads/pods/init-containers.md:138]: image

Comment (ZP-AlwaysWin) [reply content/zh/docs/concepts/workloads/pods/init-containers.md:138]: There should be something wrong with this format

Comment (tanjunchen) [reply content/zh/docs/concepts/workloads/pods/init-containers.md:138]: see 
image

Comment (tanjunchen) [reply content/zh/docs/concepts/workloads/pods/init-containers.md:138]: maybe be 
\`\`\`shell
sleep 60
\`\`\`  is much better.

**Pull request merged**